test/check/check.go

```diff
@@ -176,21 +176,21 @@ func CheckOperatorIsReady(timeout time.Duration) {
 	By("Checking that the operator is up and running")
 	if timeout != CheckImmediately {
 		Eventually(func() error {
-			return checkForDeployment(components.Name)
+			return checkForDeployment(components.Name, false)
 		}, timeout, time.Second).ShouldNot(HaveOccurred(), fmt.Sprintf("Timed out waiting for the operator to become ready"))
 	} else {
-		Expect(checkForDeployment(components.Name)).ShouldNot(HaveOccurred(), "Operator is not ready")
+		Expect(checkForDeployment(components.Name, false)).ShouldNot(HaveOccurred(), "Operator is not ready")
 	}
 }
 
 func CheckNMStateOperatorIsReady(timeout time.Duration) {
 	By("Checking that the operator is up and running")
 	if timeout != CheckImmediately {
 		Eventually(func() error {
-			return checkForGenericDeployment("nmstate-operator", "nmstate", false)
+			return checkForGenericDeployment("nmstate-operator", "nmstate", false, false)
```

Comment:
Can you elaborate why you set it to false?
UUIC `CheckNMStateOperatorIsReady` is running under `e2e/workflow` tests, so they should only run on latest code.
shouldn't we pass `true` then?

Reply:
It run installNMStateOperator which install nmstate standalone
so it doesnt need to have labels right ?

Reply:
ack